remote-api/src/main/java/org/alfresco/rest/api/model/rules/Rule.java

```diff
@@ -209,16 +217,21 @@ public void setTriggers(List<RuleTrigger> triggers)
         this.triggers = triggers;
     }
 
-    public List<Void> getActions()
+    public List<Action> getActions()
     {
-        return Collections.emptyList();
+        return actions;
+    }
+
+    public void setActions(List<Action> actions)
+    {
+        this.actions = actions;
     }
 
     @Override
     public String toString()
     {
         return "Rule{" + "id='" + id + '\'' + ", name='" + name + '\'' + ", description='" + description + '\'' + ", enabled=" + enabled + ", cascade=" + cascade
-            + ", asynchronous=" + asynchronous + ", shared=" + shared + ", errorScript='" + errorScript + '\'' + ", triggers=" + triggers + '}';
+            + ", asynchronous=" + asynchronous + ", shared=" + shared + ", errorScript='" + errorScript + '\'' + ", triggers=" + triggers + ", actions=" + actions + '}';
```

Comment:
I guess it was autogenerated because this uses the same `'\''` syntax

Reply:
The same as above - also autogenerated.